src/routes/store-config-wizard/StoreConfigWizard.tsx

```diff
@@ -1,10 +1,22 @@
+import { Container } from "@mui/system";
 import StoreConfigProvider from "./StoreConfigProvider";
 import Stepper from "./Stepper";
 
 function StoreConfigWizard() {
   return (
     <StoreConfigProvider>
-      <Stepper />
+      <Container
+        sx={{
```

Comment:
Nie widze różnicy jak usunę całe to "sx"

Reply:
removed unnecessary dields, but it is needed as it centers horizontally